### DISCUSSION TITLE
ShopAbout endpoint

### DISCUSSION THREAD

**Discussion by rajivraman** [etsy/open-api#1015] [Q&A]
Hi, I was using the ShopAbout V2 endpoint to get information about a shop, like its members and related-links. The endpoint is obviously breaking now that V2 was killed off. Any replacement for this information? Thank you.

**Comment by Gareth064** [comment]
Hey.

Have a look at https://developers.etsy.com/documentation/reference/#operation/findShops

**Reply by rajivraman** [reply]
@Gareth064 I don't see members or related-links (i.e. Facebook, Instagram, etc.) in the response of that call.

**Reply by Gareth064** [reply]
Ok. Unfortunately that is the only info you will get form v3.

You can email developer@etsy.com to ask them if they can add the other data to it.

**Reply by rajivraman** [reply]
Thank you.

**Comment by rajivraman** [comment]
I've ported this to "Ideas": https://github.com/etsy/open-api/discussions/1020
There is currently no way to get "related-links" in the V3 API
